ports/nrf/mpconfigport.h

```diff
@@ -123,15 +123,15 @@
 #define MICROPY_PY_CMATH            (0)
 #define MICROPY_PY_IO               (0)
 #define MICROPY_PY_IO_FILEIO        (0)
-#define MICROPY_PY_UERRNO           (0)
+#define MICROPY_PY_UERRNO           (1)
```

Comment:
Again, since some boards like the microbit don't have much space, these should be defined per-board rather than for all boards here.